package.json

```diff
@@ -22,32 +21,30 @@
     "url": "git://github.com/goodeggs/goodeggs-test-helpers.git"
   },
   "dependencies": {
-    "bluebird": "^3.3.4",
-    "chai": "^3.3.0",
-    "chai-as-promised": "^5.1.0",
-    "chai-datetime": "^1.4.1",
-    "chai-json-schema": "^1.4.0",
+    "chai": "^4.1.2",
+    "chai-as-promised": "^7.1.1",
+    "chai-datetime": "^1.5.0",
     "chaid": "^1.0.2",
-    "dirty-chai": "^1.2.2",
-    "lodash.isfunction": "^3.0.8",
-    "sinon": "^4.1.6",
-    "sinon-chai": "^2.7.0"
+    "dirty-chai": "^2.0.1",
+    "istanbul": "^0.4.5",
+    "mocha": "^5.0.0",
+    "mocha-multi": "^1.0.0",
+    "mocha-osx-reporter": "^0.1.2",
+    "sinon": "^4.2.2",
+    "sinon-chai": "^2.14.0"
   },
   "devDependencies": {
-    "babel-cli": "^6.6.5",
-    "babel-eslint": "^6.1.2",
-    "babel-plugin-add-module-exports": "^0.1.2",
-    "babel-plugin-transform-flow-strip-types": "^6.14.0",
-    "babel-preset-es2015": "^6.6.0",
-    "babel-register": "^6.7.2",
-    "eslint": "~3.1.1",
-    "eslint-plugin-goodeggs": "^3.3.1",
-    "eslint-plugin-lodash": "^1.10.3",
-    "eslint-plugin-mocha": "^4.5.1",
-    "flow-bin": "^0.32.0",
-    "geomoment": "^0.6.1",
-    "in-publish": "^2.0.0",
-    "mocha": "^2.4.5"
+    "babel-cli": "^6.26.0",
+    "babel-eslint": "^8.2.1",
+    "babel-plugin-add-module-exports": "^0.2.1",
+    "babel-plugin-transform-flow-strip-types": "^6.22.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-register": "^6.26.0",
+    "bunyan": "^1.8.12",
+    "eslint": "^4.16.0",
+    "eslint-plugin-goodeggs": "^6.0.0",
+    "flow-bin": "^0.64.0",
+    "geomoment": "^0.6.1"
```

Comment:
swap geomoment out with date-test-helpers while you're here? :)

Reply:
Done! 📆